scout/server/blueprints/alignviewers/views.py

```diff
@@ -50,117 +51,39 @@ def unindexed_remote_static():
 @alignviewers_bp.route("/igv", methods=["POST"])
 def igv():
     """Visualize BAM alignments using igv.js (https://github.com/igvteam/igv.js)"""
+
+    # Set genome build for displaying alignments:
+    # Genome build is 37 if request.form.get("build") is 37 and chr != MT
+    # Genome build is 38 if request.form.get("build") is 38 or if chrom == MT
+    chromosome_build = request.form.get("build")
     chrom = request.form.get("contig")
     if chrom == "MT":
         chrom = "M"
+    if chromosome_build in ["GRCh38", "38"] or chrom == "M":
+        chromosome_build = "38"
+    else:
+        chromosome_build = "37"
 
     start = request.form.get("start")
     stop = request.form.get("stop")
-
     locus = "chr{0}:{1}-{2}".format(chrom, start, stop)
-    LOG.debug("Displaying locus %s", locus)
 
-    chromosome_build = request.form.get("build")
-    LOG.debug("Chromosome build is %s", chromosome_build)
-
-    samples = request.form.get("sample").split(",")
-    LOG.debug("samples: %s", samples)
-
-    bam_files = None
-    bai_files = None
-    rhocall_bed_files = None
-    rhocall_wig_files = None
-    tiddit_coverage_files = None
-    updregion_files = None
-    updsites_files = None
-
-    if request.form.get("align") == "mt_bam":
-        bam_files = request.form.get("mt_bam").split(",")
-        bai_files = request.form.get("mt_bai").split(",")
-    else:
-        if request.form.get("bam"):
-            bam_files = request.form.get("bam").split(",")
-            LOG.debug("loading the following BAM tracks: %s", bam_files)
-        if request.form.get("bai"):
-            bai_files = request.form.get("bai").split(",")
-        if request.form.get("rhocall_bed"):
```

Comment:
But, let's see, this actually is not connected now, is it? While new and nice functions to add the tracks act if files are noted, no splitting of files from config occurs?

Reply:
You are probably right. I wanted to update this repetitive code here but I couldn't find any case with these tracks to test. Did you find one btw?

Reply:
This should be fixed now, but not tested

Reply:
Was just looking.. We had them on stage vitalmouse for the pre-MIP9 test, but there was some issue with the track formats that caused them to look dotty rather than like bar plots, so we turned them off in cg for now. Vitalmouse has probably been re-uploaded a few times since then. If stage looks calm I can try and make a test case, otw test locally.

Reply:
Agreed, the code is needlessly repetitive. I'd suggest we do all of that, including track selection in a separate PR? But lets just not disable it for now..

Reply:
I've added the splitting of the form input in https://github.com/Clinical-Genomics/scout/pull/2111/commits/5b6694d2b4e277f42488e8d0bf23b986cc8b5b5d

Reply:
Nice! Almost there, then, but the info still doesn't really propagate does it?
Setting locally
![Screenshot 2020-10-05 at 11 45 58](https://user-images.githubusercontent.com/758570/95064932-abe15d80-0700-11eb-935a-3b79db168a3a.png)
and on a master-ish branch:
![Screenshot 2020-10-05 at 11 43 58](https://user-images.githubusercontent.com/758570/95064977-bb60a680-0700-11eb-9e61-67199ceb7107.png)
but on this branch:
![Screenshot 2020-10-05 at 11 45 02](https://user-images.githubusercontent.com/758570/95065012-c61b3b80-0700-11eb-9617-372b1b03cdc3.png)

Reply:
Shouldn't the upd_regions_bed file end in .bed? Or is is just a test you're doing with no real file?

Reply:
Since when do we run DOS on the machines? ;-) But yeah, that would be the common situation. Still, works with master, but not branch?

Reply:
.txt, whatever! 😄 I'm trying!

Reply:
Thanks - don’t want to give you too much headache over this as it will anyway have to change soon again, but on the gist of not mending what ain’t broken - or not breaking what soon will need mending?

I’ll be out for lunch, but can arrange a case with some other tracks on stage later, unless you beat me to it.

Reply:
It's OK, it should work. I think I'll wait for your test case, you know better than me what it should look like! Have a nice lunch!

Reply:
I'll try to fix the code in the meanwhile

Reply:
Thanks! I'll add back the TIDDIT coverage bw that is currently on HK from MIP9. That should work on any number of cases, but I guess I'll try it on a clone of vitalmouse. I'll test your latest locally while waiting for hasta..

Reply:
Looks like it now loads locally (Still some DEBUG out left in there ofcourse.. 😄)! I didn't realise there were that many low scoring variants on the case so still loading for the clone-vital-mouse. I'll perhaps drop it and load a new one with only a few variants.

![Screenshot 2020-10-05 at 13 33 56](https://user-images.githubusercontent.com/758570/95075141-ffa77300-070f-11eb-8334-47268d2919bb.png)